src/analyses/poly.ml

```diff
@@ -20,7 +20,7 @@ struct
   let context x = if GobConfig.get_bool "exp.full-context" then x else D.bot ()
 
   let threadenter ctx lval f args = D.top ()
-  let threadspawn ctx lval f args fctx = D.bot ()
+  let threadspawn ctx lval f args fctx = ctx.local
```

Comment:
Could you explain to me why `threadspawn` should return `ctx.local` rather than `D. bot ()`?
I am bit lost here, this used to be `otherstate` iirc?

Reply:
`threadenter` used to be `otherstate`. `threadspawn` is called where the thread was created from (by any analysis).

Reply:
Thanks! Should we maybe document such things in `analyses.ml` where we define the `module type Spec`?

Reply:
I'll add that there but the entire act of spawning threads has become very nuanced. It's impossible to understand without looking at how `FromSpec` and `MCP2` put the transfer functions together and on which context which thing is called in what order. This needs a figure...